packages/sitecore-jss-react/src/components/PlaceholderCommon.tsx

```diff
@@ -211,6 +211,8 @@ export class PlaceholderCommon<T extends PlaceholderProps> extends React.Compone
 
         if (componentRendering.componentName === HIDDEN_RENDERING_NAME) {
           component = hiddenRenderingComponent ?? HiddenRendering;
+        } else if (!componentRendering.componentName) {
+          component = () => <></>;
```

Comment:
It's possible to render `null` value, did you try to set `null`?

Reply:
I think either one should work, this approach is a better fit for the code flow we have now (there's a null check below that actually causes the issue).

Reply:
Also it helps differentiate between an "empty" component (all good - means it's hidden) and "null" component (throws error, causes error frame to render).